apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ReactElement } from "react";
+
+const StopPageRedesign = ({
+  stopId
+}: {
+  stopId: string;
+}): ReactElement<HTMLElement> => {
+  return (
+    <div>
+      {/* Title Bar Div */}
+      <div className="d-flex justify-content-space-between">
+        <div>
+          <h1>{stopId}</h1>
+          {/* ICONS GO HERE */}
+        </div>
+        <div style={{ marginTop: "3.075rem" }}>
+          Zone Information PLACEHOLDER
+        </div>
+      </div>
+      {/* Route and Map Div */}
+      <div className="d-flex">
+        <div style={{ minWidth: "50%" }}>
+          <div>Route schedules & maps / Upcoming Trips PLACEHOLDER</div>
+          <div>Filter Pills PLACEHOLDER</div>
+          <div style={{ maxHeight: "250px", overflowY: "auto" }}>
+            <div className="d-flex">
+              <div className="me-8">210</div>
+              <div>
+                <div>Fields Corner</div>
+                <div className="d-flex">
+                  <div>Open Schedule</div>
+                  <div>View Realtime Map</div>
```

Comment:
non-blocking, but if you just want to show things side-by-side you don't need a `display: flex` wrapper, you can use `<span>` or any other [inline element](https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements).